joern-cli/frontends/gosrc2cpg/src/test/scala/io/joern/go2cpg/testfixtures/GoCodeToCpgSuite.scala

```diff
@@ -16,7 +16,9 @@ trait Go2CpgFrontend extends LanguageFrontend {
     val cpgOutFile = File.newTemporaryFile("go2cpg.bin")
     cpgOutFile.deleteOnExit()
     val go2cpg = new GoSrc2Cpg()
-    val config = Config()
+    val config = getConfig()
+      .map(_.asInstanceOf[Config])
```

Comment:
Does this not work? A bit safer than an unchecked cast
```suggestion
      .collectFirst { case x: Config => x }
```

Reply:
I simply mimic how it's been done at other places.

Reply:
Ok then that's fine

Reply:
I have more changes to be done for the download dependency to work completely. I will handle this one in next PR.